src/utils/helpers.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * prepends first i characters and appends last
+ * j characters of the string input to an ellipse.
+ **/
+const condenseText = (condenseThreshold, input, prefixLen, suffixLen) => {
```

Comment:
condenseThreshold should probably be the last parameter if it's optional and set it to some default (I guess 0)